packages/js-moi-logic/src.ts/state/ephemeral-state.ts

```diff
@@ -1,23 +1,79 @@
-import { ErrorCode, ErrorUtils } from "js-moi-utils";
+import { ErrorCode, ErrorUtils, hexToBytes } from "js-moi-utils";
+import { EntityBuilder } from "./entity-builder";
+import { AccessorBuilder, SlotAccessorBuilder } from "./accessor-builder";
+import { AbstractProvider } from "js-moi-providers";
+import { LogicDriver } from "../logic-driver";
+import { Schema, isPrimitiveType } from "js-moi-manifest";
+import { Depolorizer } from "js-polo";
+import { generateStorageKey } from "./accessor";
+
```

Comment:
To import tokens such as **`AccessorBuilder`**, **`LogicDriver`**, and **`AccessorBuilder`**, use **`import type`** or **`import { type NameOfImport } from "./package`**. If you use import type, it explicitly signals to the bundler that you don’t need the imported values, making tree shaking more effective.

Reply:
[7e514ee](https://github.com/sarvalabs/js-moi-sdk/pull/70/commits/7e514ee571ba04dffbb2c5224f0dcd48c6dee2e4)